core/api/service/state/impl/state_api_impl.cpp

```diff
@@ -28,6 +28,12 @@ namespace kagome::api {
     BOOST_ASSERT(nullptr != metadata_);
   }
 
+  void StateApiImpl::setApiService(
+      std::shared_ptr<api::ApiService> const &api_service) {
+    BOOST_ASSERT(api_service != nullptr);
+    api_service_ = api_service;
+  }
+
```

Comment:
I don't see the reason of the relocation of this code.

Reply:
Nothing special. Just regroup